pom.xml

```diff
@@ -147,10 +132,55 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>github-branch-source</artifactId>
+      <version>2.2.4</version>
+      <scope>test</scope>
+      <exclusions>
```

Comment:
Strange, I only followed the advice of my IDE :thinking: 

Guess it's confused, can remove exclusions :+1:

Reply:
alas the `hpi` packaging type is special and confuses many things due to its non standard hierachy of dependencies (for what actually gets packaged as a dependency)    (transitive dependencies that come from another plugin will not be packaged - but IDEs are unaware of this - similarly many tools they are unaware that depending on `someplugin v1`  where `v1` is vulnerable to a CVE is not an issue as all that the dependency is saying in the project is that v1 is a minimum version it can work with).